apps/skilavottord/ws/src/environments/environment.ts

```diff
@@ -24,5 +24,8 @@ export default {
     restUsername: process.env.FJARSYSLA_REST_USER,
     restPassword: process.env.FJARSYSLA_REST_PASS,
   },
+  skilavottord: {
+    userList: process.env.USER,
+  },
```

Comment:
This is breaking schemas.ts on main. Please pick a better env name, like "SKILAVOTTORD_USER_LIST". On many machines, the USER env is the name of the OS user, in my case "eirikur", which crashes when passed to JSON.parse().

You should also:

* Add this to the environment.prod.ts.
* Add a default value so we can generate schemas without crashing the process, eg `process.env.SKILAVOTTORD_USER_LIST ?? "[]"`.
* Document the format of this environment variable in README.md to help other devs get started with this project.

Reply:
We will fix that

Reply:
I have fixed this and changes are in main now.n